hacs.json

```
@@ -1,4 +1,5 @@
 {
   "name": "Raspberry Pi GPIO",
-  "homeassistant": "2022.7.0"
+  "homeassistant": "2022.7.0",
+  "render_readme": true
```

```suggestion
  "homeassistant": "2022.7.0"
```
`render_readme` is no longer supported by `HACS`

---

done, but dont you want to bump up the version?

---

We will bump it before the release